docs/introduction.md

```diff
@@ -1,7 +1,41 @@
 ---
 id: intro
-title: Introduction 
+title: Welcome to IOV 
 sidebar_label: Introduction 
 ---
 
-Introduction
\ No newline at end of file
+We'll point you to the docs and repos you need to get started: first, this document contains a "How to Get Started" section and further below, we have included a brief introduction to IOV for those not already familiar with the project.
+
+**IOV** is building a Universal Name Service for Wallets and Dapps.
+
+We are providing a IOV Name Service which is a register for names used in the blockchain ecosystem, such as personal value addresses, asset names, blockchain names for developers or companies. We believe naming things makes them easier to use for everyone.
+
+We are also providing a toolkit to build apps that can easily use and swap all assets from all blockchains. We believe the future of apps is there.
+
+Enjoy and please send us feedback !
+
+## How to Get Started
+
+We are building technology to **run a blockchain of names**, the IOV Name Service, and build clients for our blockchain. The code to do this is in [Weave](https://github.com/iov-one/weave "Weave Repository"). You can setup a REST API to query IOV Name Service in [BNS-proxy](https://github.com/iov-one/bns-proxy "BNS-proxy repository").You can build a client to the blockchain by using [iov-cli](https://github.com/iov-one/iov-core/blob/master/packages/iov-cli/README.md "IOV-Client Repository").
```

Comment:
Repo of the IOV Name Service API has changed: https://github.com/iov-one/iovns-rest-api